src/App.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  useEffect(() => {
+    const lowerQuery = query.toLowerCase();
+
+    setMovies(moviesFromServer.filter(movie => {
```

Comment:
if you set new state depending on the current one, you should use functional setState

Reply:
I set state depending on the data that I got from the server, not the current state.

Reply:
I see, my bad, sorry